Admin/MonitorExchangeAuthCertificate/DataCollection/Get-ExchangeAuthCertificateStatus.ps1

```diff
@@ -117,10 +118,36 @@ function Get-ExchangeAuthCertificateStatus {
 
                 if ($exchangeServersReachableList.Count -gt $currentAuthCertificateMissingOnServersList.Count) {
                     $currentAuthCertificateValidInDays = (($currentAuthCertificate.NotAfter) - (Get-Date)).Days
+
+                    if (($currentAuthCertificate.NotAfter).Date -lt (Get-Date)) {
+                        if ($currentAuthCertificateValidInDays -eq 0) {
+                            Write-Verbose ("The current Auth Certificate has expired today")
+                            $currentAuthCertificateValidInDays = -1
+                        } elseif ($currentAuthCertificateValidInDays -ne -738845) {
```

Comment:
Where is this number coming from? We should probably add in a comment about it where this number is coming from.

Reply:
Changed the logic here.